oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -1242,6 +1242,54 @@ public String getReferenceCheckpoint() {
             return referenceCp;
         }
 
+        @Override
+        public String forceIndexLaneCatchup(String confirmMessage) throws CommitFailedException {
+
+            if (!"CONFIRM".equals(confirmMessage)) {
+                String msg = "Please confirm that you want to force the lane catchup by passing 'CONFIRM' as argument";
+                log.warn(msg);
+                return msg;
+            }
+
+            if (!this.isFailing()) {
+                String msg = "The lane is not failing. This operation should only be performed if the lane is failing, it should first be allowed to catchup on its own.";
+                log.warn(msg);
+                return msg;
+            }
+
+            log.info("Running a forced catchup for indexing lane [{}]", name);
+            // First we need to abort and pause the running indexing task
+            this.abortAndPause();
+            log.info("Aborted and paused async indexing for lane [{}]", name);
+            // Release lease for the paused lane
+            this.releaseLeaseForPausedLane();
+            log.info("Released lease for paused lane [{}]", name);
+            String newReferenceCheckpoint = store.checkpoint(lifetime, ImmutableMap.of(
```

Comment:
Can we use the JDK `Map` instead of the Guava one? `Map.of(...` should be equivalent.